ecosystem/typescript/sdk_v2/src/internal/transaction.ts

```diff
@@ -7,8 +7,24 @@
 
 import { AptosConfig } from "../api/aptos_config";
 import { get } from "../client";
-import { GasEstimation } from "../types";
+import { GasEstimation, PaginationArgs, TransactionResponse } from "../types";
 import { AptosApiType } from "../utils/const";
+import { paginateWithCursor } from "../utils/paginate_with_cursor";
+
+export async function getTransactions(args: {
+  aptosConfig: AptosConfig;
+  options?: PaginationArgs;
+}): Promise<TransactionResponse[]> {
+  const { aptosConfig, options } = args;
+  const data = await paginateWithCursor<{}, TransactionResponse[]>({
+    url: aptosConfig.getRequestUrl(AptosApiType.FULLNODE),
+    endpoint: `transactions`,
+    originMethod: "getTransactions",
+    params: { start: options?.start, limit: options?.limit },
+    overrides: { ...aptosConfig.clientConfig },
```

Comment:
once https://github.com/aptos-labs/aptos-core/pull/10216/files#diff-5a5120925d00a2725ac86e969e51b41bb7cd9276bd875eb54d655d5a5f2aaae0R75, the data passed into the `paginateWithCursor()` changes

Reply:
right, right, i'll wait for that to go in before merging